docs/general/summary/help.go

```diff
@@ -0,0 +1,9 @@
+package summary
+
+var Usage = []string{"csm"}
+
+func GetDescription() string {
+	return `Generates a Summary of recorded CLI commands there were executed on the current machine.
+	The report is generated in Markdown format and saved in the root directory of JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR.
```

Comment:
...and saved in the root directory of JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR. 
--> 
...and saved in the directory stored in the JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR environment variable.

Let's make a similar correction in the description of the `saveMarkdownToFileSystem` function.